cypress/pages/BookingPage.js

```diff
@@ -0,0 +1,53 @@
+class BookingPage {
+
+    // addNameForPolishTour(){
+    //     function randomPolishTourName(){
+    //         const polishTours = ["Spływ Dunajcem", "Kraków i Wieliczka", "Szczawnica", "Energylandia", "Magiczny Kraków", "Wadowice Oświęcim" ]
+    //         var randomPolishTour = polishTours[Math.floor(Math.random()*polishTours.length)];
+    //         return randomPolishTour
+    //     }
+    //         cy.get('[id="Name"]').type(randomPolishTourName()) 
+    //     }
+
+    addNameForTour(name,priceList){  
+            cy.get('[id="Name"]').type(name) 
```

Comment:
konwencja [atrybut="nazwa"] to jest ostatecznosc,
dla id stosuj #Name
dla klas .klasaMaturalna

Wiecej info:
https://www.w3schools.com/cssreF/css_selectors.php

Reply:
so little effort required but still not fixed :(